data/help/texts.json

```diff
@@ -220,22 +220,22 @@
-      "Skilled survivors can use wrenches, hacksaws, and welding tools to add or remove parts for a vehicle.  You can even create a vehicle from scratch, by using the Construction (<press_construction>) menu to 'Start a New Vehicle' when they have a frame available.  Tuning a vehicle for speed, fuel consumption, off-road capacity, protection against threats, and cargo capacity is a complicated process.",
+      "\"Offroad\" is a rough estimate of how efficient your vehicle moves when not on pavement.  Your vehicle's safe speed, top speed, and acceleration are multiplied by roughly the offroad percentage value when your vehicle is not on the pavement.  Offroad percentage decreases as vehicle mass increases, and increases with the size and number of wheels.  Tank treads will give your vehicle an excellent offroad percentage but your vehicle will be slower due to the treads' very high rolling drag.",
+      "Skilled survivors can use wrenches, hacksaws, and welding tools to add or remove parts on a vehicle.  You can even create a vehicle entirely from scratch, by using the Construction (<press_construction>) menu to 'Start Vehicle Construction' when they have a frame available.  Tuning a vehicle for speed, fuel consumption, offroad capacity, protection against threats, and cargo capacity is a complicated process.",
       "BOATS AND AMPHIBIOUS VEHICLES",
       "You can use boat hull parts to create boats or amphibious vehicles that can move between water and land.",
-      "Each boat hull you add to your vehicle increases the height the vehicle's watertight hull.  If the height of the watertight hull is less than the vehicle's draft, then water will flow into your vehicle and it will sink and be destroyed if it goes into deep water.  If your vehicle's draft is less than the height of the watertight hull, your vehicle will not take on water and will not sink in deep water.",
-      "Draft is determined by length and width of your vehicle, the mass of your vehicle, and the percentage of the vehicle's underbody covered in boat hulls.  In general, a vehicle should be able to float if it's mass in tons is less than a quarter of the value of its length and width.  A boat that is 5 tiles wide and 9 tiles long has a good chance of floating if all of its underbody is covered in boat boards and it weighs less than 10 tons.",
+      "Each boat hull you add to your vehicle increases the height of the vehicle's watertight hull.  If the height of the watertight hull is less than the vehicle's draft, then water will flow into your vehicle and it will sink and be destroyed if it goes into deep water.  If your vehicle's draft is less than the height of the watertight hull, your vehicle will not take on water and will not sink in deep water.",
```

Comment:
```suggestion
      "Each boat hull you add to your vehicle increases the height of the vehicle's watertight hull.  If the height of the watertight hull is less than the vehicle's draft, then water will flow into your vehicle, and it will sink and be destroyed if it goes into deep water.  If your vehicle's draft is less than the height of the watertight hull, your vehicle will not take on water and will not sink in deep water.",
```